src/Components/Carousel/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import CarouselFrame from './CarouselFrame';
+import './index.scss';
+
+export default class Carousel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      listTransform: 0,
+      listTransition: '1s ease-in-out',
+    };
+  }
+
+  moveToNext = () => {
+    if (this.state.listTransform > -640 * 4) {
+      this.setState({
+        listTransform: this.state.listTransform - 640,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTransform: 0,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  moveToPre = () => {
+    if (this.state.listTransform < 0) {
+      this.setState({
+        listTransform: this.state.listTransform + 640,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTransform: -640 * 4,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      this.moveToNext();
+    }, 4000);
```

Comment:
여기도 상수 변수로 바꿔서 컴포넌트 바깥으로 주세요!
```suggestion
    }, INTERVAL);
```